ccmlib/node.py

```diff
@@ -287,20 +291,20 @@ def show(self, only_status=False, show_cluster=True):
         """
         self.__update_status()
         indent = ''.join([" " for i in xrange(0, len(self.name) + 2)])
-        print_("%s: %s" % (self.name, self.__get_status_string()))
```

Comment:
revert the changes from the `show` function

Reply:
why?

Reply:
cause those function need to print to stdout with logging

Reply:
please revert the change on this function

Reply:
Done